rules/slack_rules/slack_private_channel_made_public.yml

```diff
@@ -7,6 +7,11 @@ LogTypes:
   - Slack.AuditLogs
 Tags:
   - Slack
+  - Defense Evasion
+  - File and Directory Permissions Modification
+Reports:
+  MITRE ATT&CK:
+    - TA0005:T1222
```

Comment:
Would `TA0003:T1098` be more applicable here?
```
... Account manipulation may consist of any action that preserves or modifies adversary access to a compromised account, such as modifying credentials or permission groups. These actions could also include account activity designed to subvert security policies, such as performing iterative password updates to bypass password duration policies and preserve the life of compromised credentials.
```

Reply:
This one could fall into several categories, `T1098` makes sense, `T1562` or even `T1567` could also be applicable.